Align four 6.0 PRs (#7923)
pingcap/docs: tiflash/use-tiflash.md

* `AVAILABLE` indicates whether the TiFlash replicas of this table is available or not. `1` means available and `0` means unavailable. Once the replicas become available, this status does not change. If you use DDL statements to modify the number of replicas, the replication status will be recalculated.
* `PROGRESS` means the progress of the replication. The value is between `0.0` and `1.0`. `1` means at least one replica is replicated.

## Create TiFlash replicas for databases

> Similar with creating TiFlash replicas for tables, you can send a DDL statement to TiDB through a MySQL client to create a TiFlash replica for all tables in a specific database:

Reviewer: a DDL statement -> a SQL statement

> Similar with creating TiFlash replicas for tables, you can send a DDL statement to TiDB through a MySQL client to create a TiFlash replica for all tables in a specific database:

Author: ```suggestion
Similar to creating TiFlash replicas for tables, you can send a SQL statement to TiDB through a MySQL client to create a TiFlash replica for all tables in a specific database:
```

{{< copyable "sql" >}}

```sql
ALTER DATABASE db_name SET TIFLASH REPLICA count
```

> In this statement, `count` indicates the number of replicas. When the value is `0`, the replica is deleted.

Reviewer: Should we use `the replicas will be deleted`? Since `ALTER DATABASE` involves replica for many tables

> In this statement, `count` indicates the number of replicas. When the value is `0`, the replica is deleted.

Author: ```suggestion
In this statement, `count` indicates the number of replicas. When the value is `0`, replicas are deleted.
```

Examples:

- Create two replicas for all tables in the database `tpch50`:

    {{< copyable "sql" >}}

    ```sql
    ALTER DATABASE `tpch50` SET TIFLASH REPLICA 2
    ```

- Delete TiFlash replicas created for the database `tpch50`:

    {{< copyable "sql" >}}

    ```sql
    ALTER DATABASE `tpch50` SET TIFLASH REPLICA 0
    ```

> **Note:**
>
> - This statement actually performs a series of DDL operations, which is resource-intensive. If the statement is interrupted during the execution, executed operations are not rolled back and unexecuted operations do not continue.
>

> > - After executing the statement, do not set the number of TiFlash replicas or DDL operations on this database until **all tables in this database are replicated**. Otherwise, unexpected results may occur, which includes:

Reviewer 2: ```suggestion
> - After executing the statement, do not set the number of TiFlash replicas or perform DDL operations on this database until **all tables in this database are replicated**. Otherwise, unexpected results might occur, which include:
```

>     - If you set the number of TiFlash replicas to 2 and then change the number to 1 before all tables in the database are replicated, the final number of TiFlash replicas of all the tables is not necessarily 1 or 2.

> >     - After executing the statement, if you create tables in this database before the completion of the statement execution, TiFlash replicas **may** be created for these new tables.

Reviewer: I suggest **may or may not** be, to be more clear

> >     - After executing the statement, if you create tables in this database before the completion of the statement execution, TiFlash replicas **may** be created for these new tables.

Author: ```suggestion
>     - After executing the statement, if you create tables in this database before the completion of the statement execution, TiFlash replicas **may or may not** be created for these new tables.
```

> >     - After executing the statement, if you add indexes for tables in the database before the completion of the statement execution, the statement may be hung and resumes only after the operation of adding indexes finishes.

Reviewer 2: ```suggestion
>     - After executing the statement, if you add indexes for tables in the database before the completion of the statement execution, the statement might hang and resume only after the indexes are added.
```

>
> - This statement skips system tables, views, temporary tables, and tables with character sets not supported by TiFlash.

### Check replication progress

> Similar with creating TiFlash replicas for tables, successful execution of the statement does not mean the completion of replication. You can execute the following SQL statement to check the progress of replication on target tables:

Author: ```suggestion
Similar to creating TiFlash replicas for tables, successful execution of the statement does not mean the completion of replication. You can execute the following SQL statement to check the progress of replication on target tables:
```

> Similar to creating TiFlash replicas for tables, successful execution of the statement does not mean the completion of replication. You can execute the following SQL statement to check the progress of replication on target tables:

Reviewer 2: ```suggestion
Similar to creating TiFlash replicas for tables, successful execution of the DDL statement does not mean the completion of replication. You can execute the following SQL statement to check the progress of replication on target tables:
```

{{< copyable "sql" >}}

```sql
SELECT * FROM information_schema.tiflash_replica WHERE TABLE_SCHEMA = '<db_name>'
```

> To check tables not setting TiFlash replicas in the database, you can execute the following SQL statement:

Reviewer 2: ```suggestion
To check tables without TiFlash replicas in the database, you can execute the following SQL statement:
```
